entrypoint.sh

```diff
@@ -78,10 +78,14 @@ VIRTME_EXPECT_BOOT_TIMEOUT="300"
 
 KERNEL_SRC="${PWD}"
 
+# used to pass environment variables to the VM
+BASH_PROFILE="/root/.bash_profile"
```

Comment:
good idea!

So it will be used in both the manual and auto modes I suppose?

Reply:
I have only tested the manual mode, but from what I understand the auto modes will interact with the serial console thus will inherit this naturally.

Reply:
Thanks, I can test that!